src/api/app/models/branch_package.rb

```diff
@@ -185,7 +186,7 @@ def create_branch_packages(tprj)
            p[:package].project != p[:link_target_project]
           opts[:extendvrev] = '1'
         end
-        tpkg.branch_from(oproject, opackage, opts)
+        tpkg.branch_from(oproject, opackage, opts) unless @scmsync
```

Comment:
Why pass this state around? We know if oproject/opackage is scmsync'ed right? :)

Reply:
True